wiremock-wrapper/fetch-wiremock-and-run.py

```diff
@@ -0,0 +1,54 @@
+
+import urllib2
+import subprocess
+import os
+import sys
+
+
+def run(port):
```

Comment:
We should make the wiremock wrapper a submodule so that we can include it in the other client projects.

https://github.com/julianghionoiu/tdl-client-test-wiremock